src/barril/units/_tests/test_fixedarray.py

```diff
@@ -184,3 +184,12 @@ def testFixedArrayIndexAsScalar():
     assert fixed_array.IndexAsScalar(0).GetCategory() == "length of path"
     assert fixed_array.IndexAsScalar(0) == Scalar("length of path", 1, "m")
     assert fixed_array.IndexAsScalar(1) == Scalar("length of path", 2, "m")
+
+
+def testFixedArrayScalarOperations():
+    original = units.FixedArray(3, "length", [100, 80, 50], "m")
+    expected_division = units.FixedArray(3, "length", [50, 40, 25], "m")
+    expected_subtraction = units.FixedArray(3, "length", [-98, -78, -48], "m")
+
+    assert original / 2 == expected_division
```

Comment:
I think it is simpler to just inline the expected value

```suggestion
    assert original / 2 == units.FixedArray(3, "length", [50, 40, 25], "m")
```

Even more so when you add all operands. 

You may also consider writing a short helper to make things easier to read and maintain:

```python
def testFixedArrayScalarOperations():
    def Make3Array(values):
        return units.FixedArray(3, "length", values, "m")

    original = Make3Array([100, 80, 50])

    assert original / 2 == Make3Array([50, 40, 25])
    assert 2 - original == Make3Array([-98, -78, -48])
```

Alternatively, given this is the same "test code" using different operators, this makes it a great candidate for `parametrize`:

```python
@pytest.mark.parametrize('left, op, right, expected', [
    ([100, 80, 50], operator.truediv, 2, [50, 40, 25]),
    (2, operator.sub, [100, 80, 50], [-98, -78, -48]),
])
def testFixedArrayScalarOperations(left, op, right, expected):
    if isinstance(left, list):
        left = units.FixedArray(3, "length", left, "m")
    if isinstance(right, list):
        right = units.FixedArray(3, "length", right, "m")        

    result = op(left, right)
    assert result == expected
```

The last one is my preferred, as it allows you now to trivially add all the missing test cases. Also if a few of them fail, you get all the failures, instead of a single failure.

You can read more about parametrization here: https://docs.pytest.org/en/latest/parametrize.html